src/components/header/HeaderComponent.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="header-container">
```

Comment:
deze div heb je niet nodig voor een container, je kan beter het header element hiervoor gebruiken, is netter

Reply:
let erop dat die ook al in App.vue staat